spec/javascripts/additional_user_validation_fields_spec.js

```diff
@@ -36,30 +34,29 @@ describe('Additional User Validation Fields', function() {
     });
   });
 
-  describe('groupedUserFields()', function() {
-    beforeAll(function() {
-      this.fixtures = fixture.load('additional_user_validation_fields_elements.html');
+  describe('groupedUserFields()', () => {
+    beforeEach(() => {
+      document.body.innerHTML = fixture_elements;
     });
 
-    describe('buttonWrapper()', function() {
-      it('returns the div wrapping the button', function() {
-        expect(additionalUserValidationFields.buttonWrapper()).toExist();
```

Comment:
One of the consequences of the removal of jasmine-jquery.. Doesn't feel too bad.